extras/zui.js

```diff
@@ -1,231 +1,230 @@
-(function(Two) {
+import _ from '../src/utils/dash';
```

Comment:
zui.js and everything planning to hit `/extras/` isn't intended to be bundled with the Two.js builds. Do you recommend a way to package these modules separately?

Maybe they have a folder in the e.g: `/build/extras/`? What do you think?

Reply:
That would be doable. I'd have to rewrite ZUI to not import the lodash clone, but that's the only private Two.js API it still uses.